src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -360,6 +360,11 @@ public void rezeroLift() {
     elevatorController.setGoal(elevatorLeftEncoder.getPosition());
   }
 
+  public void zeroLiftAtCurrentPos() {
+    Cal.Lift.ARM_ABSOLUTE_ENCODER_ZERO_POS_DEG = armAbsoluteEncoder.getPosition();
+    Cal.Lift.ELEVATOR_ABS_ENCODER_POS_AT_START_INCHES = elevatorLeftAbsEncoder.getPosition();
```

Comment:
We probably want to zero these separately

Reply:
Also for `ELEVATOR_ABS_ENCODER_POS_AT_START_INCHES ` should be set to the whole thing where we take the encoder differences and multiply by a factor. `elevatorLeftAbsEncoder.getPosition()` is in degrees not inches.

```
    // Set elevator encoder position from absolute encoders
    double elevatorDutyCycleEncodersDifferenceDegrees =
        AngleUtil.wrapAngleAroundZero(
            (elevatorLeftAbsEncoder.getPosition() - elevatorRightAbsEncoder.getPosition()));
    elevatorLeftEncoder.setPosition(
        (elevatorDutyCycleEncodersDifferenceDegrees
                * Constants.Lift.ELEVATOR_MOTOR_ENCODER_DIFFERENCES_SCALAR_INCHES_PER_DEGREE)
            - Cal.Lift.ELEVATOR_ABS_ENCODER_POS_AT_START_INCHES);
```